launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -214,6 +215,8 @@
   </group>
 
   <group>
+    <let name="merger/output/objects" value="objects_before_filter" if="$(var use_vector_map)"/>
+    <let name="merger/output/objects" value="$(var output/objects)" unless="$(var use_vector_map)"/>
     <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
       <arg name="input/object0" value="temporary_merged_objects"/>
       <arg name="input/object1" value="clustering/objects"/>
```

Comment:
Is this work correctly? :pray:  
```
      <arg name="output/object" value="objects_before_filter"/>
```

I think the following code is correct
```
      <arg name="output/object" value="($var merger/output/objects)"/>
```

Reply:
Thanks,  @yukkysaito Saito-San. :pray: I tested with just lidar data. Missed this part.

Reply:
Thank you for commit